cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -1761,6 +1926,51 @@ pub mod test_helpers {
 
         client.reset_mocks(responses);
     }
+
+    /// Create a catalog with the given name and config.
+    ///
+    /// Will continue with config and not return an error if the catalog already exists.
+    pub async fn create_catalog_with_config(
```

Comment:
Reviewer note: this was copied from #3239 but is unused in this PR. It may get deleted in the followup PR if it's not used.